Source/gui/SNTBinaryMessageWindowController.h

```diff
@@ -59,6 +59,12 @@
 ///
 @property(weak) IBOutlet NSButton *dismissEventButton;
 
+///
+///   Reference to the "Check Event" button in the XIB. Used to help the user
+///   check the binary.
+///
+@property(weak) IBOutlet NSButton *checkEventButton;
+
```

Comment:
This was an accidental squash error, will replace this and move it to another PR.